app/javascript/components/mentoring/mentor_inbox/conversation_sorter.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+const sortingOptions = [
```

Comment:
Might be for another PR, but should we send this via the API as well? It allows us to introduce more sorting options server side without worrying about our JS.

Reply:
Yeah that would be a good idea for a follow-up PR. Would you include both the value and the description in the API?

Reply:
Yes. I think we should include both so that the client side code is really flexible.

Reply:
Woudl you include it in the existing API call that retrieves the conversation list or create a new API endpoint?

Reply:
Oh man. Good question. I don't know. Should we go with whats easier right now?

Reply:
I'm fine with either option :)